templates/borg_defaults.sh.erb

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+# This file is managed by Puppet.
+
+export BORG='/usr/local/bin/borg'
+export BORG_RSH='ssh -i <%= @sshdir %>/backup_id_<%= @ssh_key_type %> -o StrictHostKeyChecking=no'
+export BORG_PASSPHRASE='<%= @encryption_passphrase %>'
+export BORG_PRUNE_OPTIONS='<%= @prune_options_string %>'
+export BORG_RELOCATED_REPO_ACCESS_IS_OK=yes
+
+SERVER_LIST='<%= @server_list %>'
+# CHOOSE BACKUP SERVER THAT IS RESPONDING
+for SERVER in $SERVER_LIST; do
+  if nc -z $SERVER 22 2>/dev/null; then
```

Comment:
Suggest that a better test it to try to use borg to look at the repository. That is a more complete test to see if the server/repo is responsive.